sites/all/themes/wille/template.php

```diff
@@ -112,18 +112,26 @@ function wille_ting_collection_view_alter(&$build) {
  * Wraps the cover in a link to the material.
  */
 function wille_ting_object_cover($variables) {
+  $ding_entity_id = $variables['elements']['#object']->ding_entity_id;
+  $id_quota = $ding_entity_id . '_quota';
+  $type = $variables['meta_for_labels']['type'];
+  $id_type = $ding_entity_id . '_' . $type; 
+  $quota_explanation = ($variables['meta_for_labels']['on_quota'] ? t('A quota will be used on this material') : t('A quota will not be used on this material'));
```

Comment:
No need to wrap in `()`, i.e. just write `$quota_explanation = $variables['meta_for_labels']['on_quota'] ? t('A quota will be used on this material') : t('A quota will not be used on this material');`